src/main/java/org/olf/folio/order/util/MarcUtils.java

```diff
@@ -186,6 +191,27 @@ public String getElectronicIndicator(DataField nineEighty ) {
 		return electronicIndicator;
 	}
 	
+	public String getPublisher(DataField twoSixtyFour ) {
+        String publisher = new String();
+        if (twoSixtyFour != null) {
+            publisher = twoSixtyFour.getSubfieldsAsString(PUBLISHER);
+        } else {
+            return null;
+        }
+        return publisher;
+    }
```

Comment:
The Amazon_publisher-fail.mrc file put into slack won't download to my mac.  Can it be added as a commit to the marc-test-files directory?

Reply:
Done.

Reply:
Looking at the AmazonFO.1.mrc file I can that some of the 264 fields have a $c subfiedl (publication date) but not a $b subfield.  There are also a couple of instances with multiple $b subfields in one 245 datafield.  
For example,

264  4$c©2018
264  1$aLondon :$bKoenig Books ;$aAmsterdam :$bStichting Egress Foundation,$c[2020]
264  1$aAntwerp :$bRoyal Academy of Fine Arts Antwerp ;$aBerlin :$bb_books,$c[2021]

It also looks like the $a subfield contains the Publisher location and could be concatenated to the end of the $b subfield(s).  Note that $b subfields frequently end with a comma

Reply:
The app doesn't return a publisher for the amazon_publisher-fail.1.mrc file because the 245 datafield does not contain a $b subfield

264  4$c©2020

Reply:
@fereira Publisher is mapped to the `264$b`, correct? Here's what I see as the value for `amazon_publisher-fail.1.mrc`:

```
=264  \1$aParis :$bAfter 8 Books,$c[2020]
```

Reply:
Unless otherwise specified, the app assumes there is only one instance of a Datafield.  For the list of identifiers and contributors (and when it used to read the 856 fields) , it does read multiple Datafields of the same type, but no others. I can modify the code so that it reads multiple 264 fields but only one Publisher string can be added to the orderLine.  If I read the Datafield, and use the getIndicator2 method, one returns a 4 and the other a 1.  I don't know what the value indicator2 means, but it looks like if it has a indicator2 to value = 1 than it contains $b subfield, and a 264 file with an indicator2 value = 4 does not.

Reply:
Thanks for confirming the app behavior.

I'm way out of my element here, but looking at the [LOC MARC documentation](https://www.loc.gov/marc/bibliographic/bd264.html), I think we should ignore `264` fields with `4` in the second indicator when mapping the publisher. Should we only use the `264` with a `1` in the second indicator? That's a question for @starsplatter or @natalyapik.

Reply:
I think that is right.

Reply:
I was just looking at the documentation and I agree.  I don't *think* it will be an issue but I think there will only be one 264 tag with a 2nd indicator=1 per record.  If there *is* more than one, we can just return the first one.  Based on how the $a, $b, and $c subfields are formatted, a concatentation of $b + $a + $c would produce a Publisher string like this: After 8 Books, Paris : [2020] (note the comma and colon as part of the strings).

Reply:
at the point of load records still could be very preliminary, so we should ignore indicators and use what it is in 264 regardless, i think, and yes there should be only one 264.

Reply:
the problem with that, if there are multiple 264 fields (and there are with some of our test data)  we have to be to determine which one to use, and the indicator2 field with a value of 4, according to the documentation (https://www.loc.gov/marc/bibliographic/bd264.html) , indicates that the field only contains a copyright notice date.  We can't ignore indicators if we want to make sure that the data we're getting from the 264 is for a Publisher

Reply:
That makes sense. I think for this we just want to pull in the publisher.

Reply:
Summarizing the consensus for everyone's benefit:

* map `publisher` to the first `264 \1 $b`
* no concatenation of other subfields

Reply:
latest commits have resolved this